geopandas/tools/_print_versions.py

```diff
@@ -0,0 +1,142 @@
+import codecs
+import importlib
+import locale
+import os
+import platform
+import struct
+import subprocess
+import sys
+
+
+def get_sys_info():
+    "Returns system information as a dict"
+
+    blob = []
+
+    # get full commit hash
+    commit = None
+    if os.path.isdir(".git") and os.path.isdir("pandas"):
+        try:
+            pipe = subprocess.Popen('git log --format="%H" -n 1'.split(" "),
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.PIPE)
+            so, serr = pipe.communicate()
+        except (OSError, ValueError):
+            pass
+        else:
+            if pipe.returncode == 0:
+                commit = so
+                try:
+                    commit = so.decode('utf-8')
+                except ValueError:
+                    pass
+                commit = commit.strip().strip('"')
+
+    blob.append(('commit', commit))
+
+    try:
+        (sysname, nodename, release,
+         version, machine, processor) = platform.uname()
+        blob.extend([
+            ("python", '.'.join(map(str, sys.version_info))),
+            ("python-bits", struct.calcsize("P") * 8),
+            ("OS", "{sysname}".format(sysname=sysname)),
+            ("OS-release", "{release}".format(release=release)),
+            # ("Version", "{version}".format(version=version)),
+            ("machine", "{machine}".format(machine=machine)),
+            ("processor", "{processor}".format(processor=processor)),
+            ("byteorder", "{byteorder}".format(byteorder=sys.byteorder)),
+            ("LC_ALL", "{lc}".format(lc=os.environ.get('LC_ALL', "None"))),
+            ("LANG", "{lang}".format(lang=os.environ.get('LANG', "None"))),
+            ("LOCALE", '.'.join(map(str, locale.getlocale()))),
+        ])
+    except (KeyError, ValueError):
+        pass
+
+    return blob
+
+
+def show_versions(as_json=False):
+    """
+    Print system information and installed module versions.
+
+    Example
+    -------
+    > python -c "import geopandas as gpd; gpd.show_versions()"
```

Comment:
Can you use `import geopandas` and then `geopandas.show_versions()` in the example for consistency of docs?